moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -17,17 +17,19 @@ import {
   DELETE_WORK_ACTIVITY,
 } from "../../../../queries/funding";
 import { currencyFormatter } from "src/utils/numberFormatters";
+import { useHiddenColumnsSettings } from "src/utils/localStorageHelpers";
 
 /** Hook that provides memoized column settings */
 const useColumns = ({ deleteInProgress, onDeleteActivity, setEditActivity }) =>
   useMemo(() => {
     return [
       {
         headerName: "",
-        field: "_edit",
+        field: "Edit",
```

Comment:
the other fields are lower case, it just jumped out at me while I was adding pizza to the localStorage

Reply:
ah, this is because this is the way the field is labeled in the column picker. i guess i should just give it a header name 🤔